integrations/prodigy_ner_fashion/assets/fashion_brands_patterns.jsonl

```diff
@@ -0,0 +1,100 @@
+{"label":"FASHION_BRAND","pattern":[{"lower":"ann"},{"lower":"taylor"}]}
```

Comment:
Let's remove the patterns as well, since they're not relevant for this specific example template.